docs/modules/ROOT/pages/how-tos/upgrade-1.1.1-1.7.x.adoc

```diff
@@ -0,0 +1,9 @@
+= Upgrade from 1.1.x to 1.2.x
+
+This guide describes the steps to perform an upgrade of the component from version 1.1.x to 1.2.x.
+
+== Step-by-step guide
+
+Component version 1.2.x installs Crossplane version 1.7.1. Since Crossplane Version 1.3, all CRDs are being managed by the Crossplane Operator and no longer by the Helm Chart. An automated PreSync Hook during migration will ensure that CRDs aren't deleted.
+
+NOTE: ALl CRDs are expected to use api version _v1_ (stored version) while on component version _1.1.1_ with exception to Locks and ControllerConfig. The Lock object is no longer managed by the HelmChart in Crossplane 1.7 therefore it will be ignored during migration by ArgoCD. ControllerConfig CRD hasn't received any updates.
```

Comment:
```suggestion
[NOTE]
====
 All CRDs are expected to use api version _v1_ (stored version) while on component version _1.1.1_ with exception to Locks and ControllerConfig.
 The Lock object is no longer managed by the HelmChart in Crossplane 1.7 therefore it will be ignored during migration by ArgoCD.
 ControllerConfig CRD hasn't received any updates.
 ====
```